k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -72,10 +72,6 @@
         MessageViewFragmentListener, OnBackStackChangedListener, OnSwipeGestureListener,
         OnSwitchCompleteListener {
 
-    @Deprecated
-    //TODO: Remove after 2017-09-11
-    private static final String EXTRA_SEARCH_OLD = "search";
```

Comment:
I am not sure there's value in removing this. I think we should remove the comment instead.

Reply:
See comment below: https://github.com/k9mail/k-9/pull/2909#discussion_r150975614

Reply:
@philipwhiuk: Removing this shouldn't be a problem. There's only a short window where this could lead to a problem. That's when updating from an app version using the old format to a version not supporting it and having an unread widget with state from the old installation. As soon as the widget is updated with new state it'll use the correct Intent extra.